tests/components/google_generative_ai_conversation/test_conversation.py

```diff
@@ -211,7 +211,7 @@ async def test_function_call(
             name="test_tool",
             args={
                 "param1": ["test_value", "param1\\'s value"],
-                "param2": "param2\\'s value",
```

Comment:
Why do you need to change this test? I added it when I fixed controlling entities with ' in their name, e.g. Alice's room.

Reply:
The `param2` is defined in its schema as `vol.Any(int, float)` so a string value doesn't pass validation.

Reply:
We already have an `'` in `param1`, but if it is not enough, I can add another string parameter